src/ext.js

```diff
@@ -336,6 +352,14 @@ export default function ext({ translator, shouldHide, senseNavigation }) {
                 sizeAndColor: {
                   type: 'items',
                   items: {
+                    fontSizeBehavior: {
+                      component: 'dropdown',
+                      ref: 'style.font.sizeBehavior',
+                      options: fontSizeOptions,
+                      translation: 'properties.kpi.layoutBehavior',
```

Comment:
A bit ugly because it uses an KPI translation. It probably should not have been a specific KPI translation from the start.